test/utils/UtilsSpec.scala

```diff
@@ -48,4 +54,137 @@ class UtilsSpec extends SpecBase {
       singleSpace mustBe " "
     }
   }
+
+  "negativeSign" should {
+    "return correct value" in {
```

Comment:
either add a new line between the should and the in or remove them from the other tests

Reply:
Done